packages/devtools_app/lib/src/screens/vm_developer/vm_object_model.dart

```diff
@@ -8,18 +8,24 @@ import 'package:vm_service/vm_service.dart';
 import '../../service/vm_service_wrapper.dart';
 import '../../shared/globals.dart';
 import '../debugger/debugger_model.dart';
+import '../debugger/program_explorer_model.dart';
 
 /// Wrapper class for storing Dart VM objects with their relevant VM
 /// information.
 abstract class VmObject {
-  VmObject({required this.ref});
+  VmObject({required this.ref, this.scriptRef, this.outlineNode});
 
   final ObjRef ref;
 
+  final ScriptRef? scriptRef;
```

Comment:
Can you add comments for these new properties explaining what they represent?

Reply:
Done